src/components/FoldedListbox.tsx

```diff
@@ -12,14 +12,12 @@ export const FoldedListbox = ({ layout }: FoldedListboxProps) => {
   const fieldName = useFieldName(layout);
 
   return (
-    <>
-      <Typography
-        variant="h6"
-        border="1px solid lightgrey"
-        padding="8px"
-      >
-        {fieldName}
-      </Typography>
-    </>
+    <Typography
+      variant="h6"
+      border="1px solid lightgrey"
+      padding="8px"
```

Comment:
And for this I think `padding={1}` is equivalent of `8px`: https://mui.com/system/getting-started/the-sx-prop/#spacing – don't need to change, just mentioning.